src/main/java/com/google/sps/servlets/SentimentServlet.java

```diff
@@ -0,0 +1,92 @@
+package com.google.sps.servlets;
+
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import com.google.sps.data.VideoAnalysis;
+import java.io.IOException;
+import java.util.List;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/sentiment")
+public class SentimentServlet extends HttpServlet {
+
+  private LanguageServiceClient languageService;
+  private Sentiment sentiment;
+
+  public SentimentServlet() throws IOException {
+    languageService = LanguageServiceClient.create();
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Float commentsScore = null;
+    boolean isValid = true;
+
+    try {
+      String comments = requestToString(request);
+      if (comments.equals("")) {
+        response.sendError(HttpServletResponse.SC_BAD_REQUEST,
+            "No comments to analyze. Request must be a non-empty array");
+        isValid = false;
```

Comment:
doGet is a void method (and also overrides a method of its superclass so I can't change it), so I used isValid to avoid running the rest of the method if there's an error.

Reply:
nvm, Zoe helped me out with this!